src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -187,40 +296,43 @@ async Task<int> Read(byte[] buffer, int offset, int count, bool async)
                     throw;
                 }
 
-                switch (msg.Code) {
-                case BackendMessageCode.CopyData:
-                    _leftToReadInDataMsg = ((CopyDataMessage)msg).Length;
-                    break;
-                case BackendMessageCode.CopyDone:
-                    Expect<CommandCompleteMessage>(await _connector.ReadMessage(async), _connector);
-                    Expect<ReadyForQueryMessage>(await _connector.ReadMessage(async), _connector);
-                    _isConsumed = true;
-                    return 0;
-                default:
-                    throw _connector.UnexpectedMessageReceived(msg.Code);
+                switch (msg.Code)
+                {
+                    case BackendMessageCode.CopyData:
+                        _leftToReadInDataMsg = ((CopyDataMessage)msg).Length;
+                        break;
+                    case BackendMessageCode.CopyDone:
+                        Expect<CommandCompleteMessage>(await _connector.ReadMessage(async), _connector);
+                        Expect<ReadyForQueryMessage>(await _connector.ReadMessage(async), _connector);
+                        _isConsumed = true;
+                        return 0;
+                    default:
+                        throw _connector.UnexpectedMessageReceived(msg.Code);
                 }
             }
 
             Debug.Assert(_leftToReadInDataMsg > 0);
 
             // If our buffer is empty, read in more. Otherwise return whatever is there, even if the
             // user asked for more (normal socket behavior)
-            if (_readBuf.ReadBytesLeft == 0) {
+            if (_readBuf.ReadBytesLeft == 0)
```

Comment:
We mostly omit braces for single-line conditions

Reply:
OK.